dev/ci/ci-basic-overlay.sh

```diff
@@ -79,166 +79,166 @@ project stdpp "https://gitlab.mpi-sws.org/iris/stdpp" ""
 # HoTT
 ########################################################################
-project hott "https://github.com/HoTT/HoTT" "master"
+project hott 'https://github.com/HoTT/HoTT' 'd79c4bb3c0a10ceb8d8ff2acd4ffc4c645ffd5c9'
 
 ########################################################################
 # CoqHammer
 ########################################################################
-project coqhammer "https://github.com/lukaszcz/coqhammer" "master"
+project coqhammer 'https://github.com/lukaszcz/coqhammer' 'f8bd70509365f6e2c8038b773332eef935ca87ee'
 
 ########################################################################
 # GeoCoq
 ########################################################################
-project geocoq "https://github.com/GeoCoq/GeoCoq" "master"
+project geocoq 'https://github.com/GeoCoq/GeoCoq' '25917f56a3b46843690457b2bfd83168bed1321c'
 
 ########################################################################
 # Flocq
 ########################################################################
-project flocq "https://gitlab.inria.fr/flocq/flocq" "flocq-3"
+project flocq 'https://gitlab.inria.fr/flocq/flocq' 'e68658cb2e6ed16c79fc4b01d578e79052f45bf8'
 
 ########################################################################
 # coq-performance-tests
 ########################################################################
-project coq_performance_tests "https://github.com/coq-community/coq-performance-tests" "master"
+project coq_performance_tests 'https://github.com/coq-community/coq-performance-tests' '5cc9a158e3aa32bc39716100e575f5f30cc72008'
 
 ########################################################################
 # coq-tools
 ########################################################################
-project coq_tools "https://github.com/JasonGross/coq-tools" "master"
+project coq_tools 'https://github.com/JasonGross/coq-tools' 'd87641a8f0b19399c6a709796b62ae303dc11ac1'
 
 ########################################################################
 # Coquelicot
 ########################################################################
-project coquelicot "https://gitlab.inria.fr/coquelicot/coquelicot" "master"
+project coquelicot 'https://gitlab.inria.fr/coquelicot/coquelicot' '09eb630f233c6887bef8f6764c7145bc45f24951'
 
 ########################################################################
 # Gappa stand alone tool
 ########################################################################
-project gappa_tool "https://gitlab.inria.fr/gappa/gappa" "master"
+project gappa_tool 'https://gitlab.inria.fr/gappa/gappa' '6c97a36257369d89ff32c9877c0e83681bfd3df9'
 
 ########################################################################
 # Gappa plugin
 ########################################################################
-project gappa "https://gitlab.inria.fr/gappa/coq" "master"
+project gappa 'https://gitlab.inria.fr/gappa/coq' '45c464f726a7361201f4902bfcbb0aaf46b15e3d'
```

Comment:
This is the overlay commit, it will succeed because archive download works for commits from forks.